modules/islandora_scholar_embargo/islandora_scholar_embargo.module

```diff
@@ -254,8 +254,8 @@ EOQ;
       $expiry_emails[$params['pid']]['params']['dsid'] = array_merge($expiry_emails[$params['pid']]['params']['dsid'], $params['dsid']);
     }
 
-    if (isset($params['dsid']) && isset($item[$dsid])) {
-      rules_invoke_event('islandora_scholar_datastream_embargo_expiring', $item, $item[$dsid]);
+    if (isset($params['dsid'])) {
+      rules_invoke_event('islandora_scholar_datastream_embargo_expiring', $item, $params['dsid']);
```

Comment:
Not sure this is valid... it's expecting an `islandora_datastream`: https://github.com/Islandora/islandora_scholar/blob/ab835969c995be48bb5d3e44268bcb37c686a008/modules/islandora_scholar_embargo/islandora_scholar_embargo.rules.inc#L72